.github/dockerfiles/ov_build/ubuntu_22_04_x64/Dockerfile

```
@@ -86,3 +90,11 @@ RUN mkdir -p $NVM_DIR
 RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
 RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}
 ENV PATH="$NVM_DIR/versions/node/v${NODE_VERSION}/bin/:${PATH}"
+
+# Install doxygen
+ENV DOXYGEN_HOME="/opt/doxygen"
+ENV DOXYGEN_VERSION='1.9.6'
+RUN mkdir -p ${DOXYGEN_HOME} && cd ${DOXYGEN_HOME} && wget https://www.doxygen.nl/files/doxygen-$DOXYGEN_VERSION.linux.bin.tar.gz && \
+    tar -I pigz -xf doxygen-$DOXYGEN_VERSION.linux.bin.tar.gz
```

Just to keep the image clean
```suggestion
    tar -I pigz -xf doxygen-$DOXYGEN_VERSION.linux.bin.tar.gz && \
    rm -f doxygen-$DOXYGEN_VERSION.linux.bin.tar.gz
```